src/frontend/web_application/src/scenes/MessageList/index.js

```diff
@@ -0,0 +1,30 @@
+import { createSelector } from 'reselect';
+import { bindActionCreators, compose } from 'redux';
+import { connect } from 'react-redux';
+import { withTranslator } from '@gandi/react-translate';
+import { requestMessages, updateMessage } from '../../store/modules/message';
+import Presenter from './presenter';
+
+const messagesSelector = state => state.message.messages;
+const messageByIdSelector = state => state.message.messagesById;
```

Comment:
I'm agree with you, this sounds enough to work in most of my cases and I always be aware of that.

But I prefer to keep this as it is since it is recommended by the [documentation](http://redux.js.org/docs/basics/Reducers.html#note-on-relationships) and since I don't have all of use cases in mind, for example I would like to keep an ordered list in state.